changelog.md

```diff
@@ -0,0 +1,148 @@
+# Change Log
+
+## Unreleased
```

Comment:
oh really good point @jsamoocha So... i will open an issue here.. because it think what you are pointing out is that we need to discuss what our workflow will be in terms of changelog updates and release. 

Github does have an automate changelog option when you create a release. HOWEVER for us, this repo has never had a release! so i for our first release we will have to create  the changelog.  after that we can use the github release option but that depends upon really good commit messages.  let me open an issue so we can discuss. as i was thinking my next PR would be updating the README, Contributing and creating a dev guide with all of this information. but we need to decide on what our process is first.

Reply:
Good idea to discuss this in a new issue. Can we add all the known merged PRs mentioned above to the current changelog for a future 0.10.5 release?

```suggestion
## Unreleased

* Fix get_athlete w new attrs for shoes given strava updates to API (@lwasser, #220)
* Refactor deprecated unittest aliases for Python 3.11 compatibility (@tirkarthi, #223)
* Patch: Update readme and fix broken links in docs (@lwasser, #229)
* Improved unknown time zone handling (@jsamoocha, #242)
```

Reply:
Absolutely. thank you @jsamoocha one note - in my previous packages i've used issue numbers but i think here we are using pr's so i'll update mine addition AND i'll also update the docs that will be pushed in another pr! thank you for making these changes!